README.md

```diff
@@ -158,11 +159,11 @@ nix-build -A packages.x86_64-linux.jak-asan # package with Clang ASan build
 
 ### Required Software
 
-We primarily use Visual Studio for development on Windows for C++ development.  Download the community addition from [here](https://visualstudio.microsoft.com/vs/)
+We primarily use Visual Studio on Windows for C++ development.  Download the community edition from [here](https://visualstudio.microsoft.com/vs/)
```

Comment:
might wanna specify 2022 perhaps?

Reply:
I'll specify the latest release so we won't have to update the docs and just make sure it functions on the latest version.